docs/core/config/auth/databases/ldap.md

```diff
@@ -240,9 +259,9 @@ this will create a symlink pointing to the CA.
 
 You can test the CA installation with this command:
 
-```sh
-openssl s_client -connect yourldap.example.org:636 \
-  -CApath /etc/openldap/certs -showcerts
+```console
```

Comment:
Karl has been changing console -> sh, and you're changing sh -> console back. This is the only console one now.